packages/components/stepper/src/useStepper.ts

```diff
@@ -0,0 +1,37 @@
+import { useNumberField } from '@react-aria/numberfield'
+import { useNumberFieldState } from '@react-stately/numberfield'
+
+import type { UseStepperArgs, UseStepperReturn } from './types'
+
+export const useStepper = ({
+  inputRef,
+  locale = 'fr',
+  ...rest
+}: UseStepperArgs): UseStepperReturn => {
+  const state = useNumberFieldState({
+    ...rest,
+    isDisabled: rest.disabled,
+    isReadOnly: rest.readOnly,
+    isRequired: rest.required,
+    locale,
+  })
+
+  const { groupProps, inputProps, incrementButtonProps, decrementButtonProps } = useNumberField(
```

Comment:
Personal preference, but on other components we expose the whole api returned by the library (wether it is Zag, PopperJS, etc), it avoids destructuring, simplifies typing (eliminated the need to use `Pick<>` and allows us to use/explore the full API in each part of the compound.

WDYT ?

Reply:
Passing a huge and obscure pack of props could also be less readable I think. I guess it is a matter of personal preference but using (and exposing) only the necesary parts also allows us to keep our code more easy to read and understand.